<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGLFrameworkClass</class>
 <widget class="QMainWindow" name="QtGLFrameworkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGLFramework</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,0">
    <item>
     <layout class="QHBoxLayout" name="glcontainter">
      <property name="spacing">
       <number>1</number>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="controlPanel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="renderGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>114</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>130</height>
          </size>
         </property>
         <property name="title">
          <string>Render mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="ctrPnlRFill">
            <property name="text">
             <string>Fill</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ctrPnlRLine">
            <property name="text">
             <string>Line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ctrPnlRFilewire">
            <property name="text">
             <string>Fillwire</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ctrPnlFlat">
            <property name="text">
             <string>Flat</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mFOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Render mode</string>
    </property>
    <addaction name="actionFull_Screen"/>
    <addaction name="separator"/>
    <addaction name="mRFill"/>
    <addaction name="mRLine"/>
    <addaction name="mRFillWire"/>
    <addaction name="mRFlat"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <addaction name="actionOpen_control_panel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <action name="mFOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="mRFill">
   <property name="text">
    <string>Fill</string>
   </property>
  </action>
  <action name="mRLine">
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="mRFillWire">
   <property name="text">
    <string>FillWire</string>
   </property>
  </action>
  <action name="actionOpen_control_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open control panel</string>
   </property>
  </action>
  <action name="actionClose_control_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close control panel</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="icon">
    <iconset resource="qtglframework.qrc">
     <normaloff>:/QtGLFramework/Resources/20130710010820632_easyicon_net_256.png</normaloff>:/QtGLFramework/Resources/20130710010820632_easyicon_net_256.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="mRFlat">
   <property name="text">
    <string>Flat</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtglframework.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen_control_panel</sender>
   <signal>triggered(bool)</signal>
   <receiver>controlPanel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>819</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>QtGLFrameworkClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
